scene/resources/default_theme/default_theme_builders.py

```diff
@@ -0,0 +1,41 @@
+"""Functions used to generate source files during build time
+
+All such functions are invoked in a subprocess on Windows to prevent build flakiness.
+
+"""
+import os
+import os.path
+from platform_methods import subprocess_main
+
+
+def make_fonts_header(target, source, env):
+
```

Comment:
That line could be removed I guess (I'm surprised `black` is not opinionated on this?).